contributing_to_docs/term_glossary.adoc

```diff
@@ -95,7 +95,7 @@ You can host a WordPress application on OpenShift.
 
 Usage: Assisted Installer
 
-In Red Hat OpenShift, the Assisted Installer is an installation solution that is offered on the Red Hat Hybrid Cloud Console to provide Software-as-a-Service functionality for cluster installations.  
+In Red Hat OpenShift, the Assisted Installer is an installation solution that is offered on the Red Hat Hybrid Cloud Console to provide Software-as-a-Service functionality for cluster installations.
```

Comment:
we should mentioned Assisted Installer is for connected deployments.

Reply:
Added:

> The Assisted Installer supports installing {product-title} on premises in a connected environment.

Reply:
How about `Assisted Installer is recommended for connected deployments`?